go/coinstacks/cosmos/api/handlers.go

```diff
@@ -117,3 +117,12 @@ func (h *Handler) GetTxHistory(pubkey string, page int, pageSize int) (api.TxHis
 func (h *Handler) SendTx(hex string) (string, error) {
 	return h.httpClient.BroadcastTx([]byte(hex))
 }
+
+func (h *Handler) GetGasEstimation(hex string) (api.Gas, error) {
+
+	res, err := h.httpClient.GetGasEstimation([]byte(hex))
+	if err != nil {
+		return "", err
+	}
+	return api.Gas(res), nil
```

Comment:
Should be able to just return the results of `EstimateGas` which matches the return value of this handler. I also am not a big fan of type aliases, I think we can just return `string` vs `api.Gas`

```suggestion
	return h.httpClient.GetGasEstimation([]byte(hex))
```

Reply:
I didn't include in suggested change, but there should be no need to cast the `base64` string to byte slice only to cast it back to decode later.